luigi/parameter.py

```diff
@@ -917,7 +928,7 @@ def normalize(self, x):
         :param str x: the value to parse.
         :return: the normalized (hashable/immutable) value.
         """
-        return tuple(x)
+        return _recursively_freeze(x)
```

Comment:
this should still be `tuple()` I think, or do we say anywhere a "bigger" structure is allowed?